### PR TITLE
Rename `batchInsert()` to `insertBatch()` and change paremeters order

### PR DESCRIPTION
**Pull request by Tigrov**
* Rename `batchInsert()` to `insertBatch()` in `DMLQueryBuilderInterface` and `CommandInterface`
* Change parameters order to from `$table, $columns, $rows` to `$table, $rows, $columns`
* Add default value `[]` for `$columns` parameter

From `batchInsert(string $table, array $columns, iterable $rows)`
to `insertBatch(string $table, iterable $rows, array $columns = [])`

### Related PRs
* yiisoft/db-sqlite#301
* yiisoft/db-mysql#334
* yiisoft/db-pgsql#344
* yiisoft/db-oracle#268
* yiisoft/db-mssql#302

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | #772

### CODE REVIEW

File: src/Command/AbstractCommand.php [hunk 1]
```diff
@@ -193,7 +196,19 @@ public function alterColumn(string $table, string $column, ColumnInterface|strin
         return $this->setSql($sql)->requireTableSchemaRefresh($table);
     }
 
+    /**
+     * @param string[] $columns
+     *
+     * @psalm-param BatchValues $rows
+     *
+     * @deprecated Use (@see insertBatch()) instead. It will be removed in version 3.0.0.
```
Comment (vjik) [inline src/Command/AbstractCommand.php:204]: After merge PR need to create new issue about remove it.

File: src/QueryBuilder/AbstractDMLQueryBuilder.php [hunk 2]
```diff
@@ -62,7 +63,20 @@ public function __construct(
     ) {
     }
 
+    /**
+     * @param string[] $columns
+     *
+     * @psalm-param BatchValues $rows
+     * @psalm-param ParamsType $params
+     *
+     * @deprecated Use (@see insertBatch()) instead. It will be removed in version 3.0.0.
```
Comment (vjik) [inline src/QueryBuilder/AbstractDMLQueryBuilder.php:72]: After merge PR need to create new issue about remove it.

**Review by vjik — CHANGES_REQUESTED** [review]
Add info to UPGRADE.md

**Comment by Tigrov** [comment]
> Add info to UPGRADE.md and changelog

Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,8 @@
 - Enh #806: Build `Expression` instances inside `Expression::$params` when build a query using `QueryBuilder` (@Tigrov)
 - Enh #766: Allow `ColumnInterface` as column type. (@Tigrov)
 - Bug #828: Fix `float` type when use `AbstractCommand::getRawSql()` method (@Tigrov)
+- Enh #829: Rename `batchInsert()` to `insertBatch()` in `DMLQueryBuilderInterface` and `CommandInterface`
+  and change parameters order from `$table, $columns, $rows` to `$table, $rows, $columns = []` (@Tigrov)
```
Comment (vjik) [inline CHANGELOG.md:13]:
```suggestion
  and change parameters from `$table, $columns, $rows` to `$table, $rows, $columns = []` (@Tigrov)
```

**Review by vjik — APPROVED** [review]

**Pull request merged**